contracts/test/helpers.js

```diff
@@ -223,8 +224,11 @@ const setOracleTokenPriceUsd = async (tokenSymbol, usdPrice) => {
   const tokenFeed = await ethers.getContract(
     "MockChainlinkOracleFeed" + tokenSymbol
   );
-  await tokenFeed.setDecimals(8);
-  await tokenFeed.setPrice(parseUnits(usdPrice, 8));
+  const oracleRouter = await ethers.getContract("OracleRouter");
+
+  await tokenFeed.setDecimals(18);
+  await oracleRouter.cacheDecimals(tokenAddress);
+  await tokenFeed.setPrice(parseUnits(usdPrice, 18));
```

Comment:
I don't think we want all Mock oracles to be 1e18? Since many are actually 1e8? Shouldn't we just have some each way?

Reply:
good point, addressed here: https://github.com/OriginProtocol/origin-dollar/pull/1331/commits/907f2f249895f792bffa8495c9192d7f27851ff7